### PR TITLE
Clarification: API access

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: vipps-report-api-faq.md [hunk 1]
```diff
@@ -36,6 +35,29 @@ API before summer 2023.
...
+## What are the benefits of the Report API over the SFTP service?
+
+The
+[Vipps SFTP Report Service](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/settlements/sftp-report-service)
+lets merchants and partners download settlement files by SFTP.
+
+The Report API has some benefits over this:
+* Data can be retrieved at any time.
+* Data is available regardless of whether the balance is positive or negative,
+  so even if there has only been refunds there will be data available.
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:47]:
```suggestion
  so even if there have only been refunds, there will be data available.
```

File: vipps-report-api-faq.md [hunk 2]
```diff
@@ -36,6 +35,29 @@ API before summer 2023.
...
+* The SFTP service only provides settlement files if the balance is positive.
+* It is possible to retrieve data for more than one day in one request, so
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:49]: I didn't like using "it is possible" or "in one request" twice in the same sentence. You could perhaps say: "It is possible to retrieve data for more than one day in a request, so you could, for example, retrieve an entire weekend, week, or month."

File: vipps-report-api-faq.md [hunk 3]
```diff
@@ -36,6 +35,29 @@ API before summer 2023.
...
+  it is possible to retrieve an entire weekend, week, month, etc. in one request.
+* The SFTP service only provides one file per day.
+* The data is in JSON format, making it easy to use it in many different ways.
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:52]:
```suggestion
* The data is in JSON format, making it easy to use in many different ways.
```

File: vipps-report-api-faq.md [hunk 4]
```diff
@@ -123,15 +124,26 @@ The `sincePayout` works in a similar way.
 an amount is always withheld to compensate for high risk, etc. Due to this, each
 payment may not always be connected to a specific payout.
 
+## How can I get access to the API?
+
+The API is available for sale units that already have access to a
+[Vipps API](https://vippsas.github.io/vipps-developer-docs/docs/APIs).
+
+If you do not already have API access, you can not get access to the Report API.
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:132]:
```suggestion
If you do not already have API access, you cannot get access to the Report API.
```

File: vipps-report-api-faq.md [hunk 5]
```diff
@@ -123,15 +124,26 @@ The `sincePayout` works in a similar way.
...
+This is the case if you only have a "Vippsnummer" sale unit, since those
+sale units have no API access.
+
 ## Which API keys give access to the API?
 
 For now: The following API keys give access to the Report API:
 
-* The merchant's own API keys: The same API keys that are used to make payments.
+* The merchant's own API keys: The same API keys that are used to make payments, etc. See
+  [Vipps APIs]([Vipps API](https://vippsas.github.io/vipps-developer-docs/docs/APIs).
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:141]: Maybe you want to use the API keys URL instead?
https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/common-topics/api-keys

File: vipps-report-api-faq.md [hunk 6]
```diff
@@ -123,15 +124,26 @@ The `sincePayout` works in a similar way.
...
 
 Later (we can not give an ETA, but will update here):
 
-* Partner keys: The same API keys that are used to make payments.
+* [Partner keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/partner-keys):
+  The same API keys that are used by partner to make payments on behalf of their merchants.
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:146]:
```suggestion
  The API keys that are used by partners to make payments on behalf of their merchants.
```

File: vipps-report-api-faq.md [hunk 7]
```diff
@@ -142,7 +154,7 @@ Later (we can not give an ETA, but will update here):
 The merchant's own API keys give full access to the Report API.
 
 If the merchant uses an integration partner (see
-[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner#partner-types)),
+[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types)),
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:157]:
```suggestion
[partner types](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner#partner-types)),
```

Comment (rebekaburnett) [reply vipps-report-api-faq.md:157]: You can't add that slash there, Christian. It is one of the ones that gives a 404 :-0

File: vipps-report-api-faq.md [hunk 8]
```diff
@@ -167,7 +179,7 @@ Partner keys let a partner make payments on behalf of a merchant, and the same A
 also give access to the Report API.
 
 If the merchant has a sale unit that is configured with a
-[platform partner](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner#partner-types),
+[platform partner](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/#partner-types),
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:182]: same slash problem as above.

File: vipps-report-api-faq.md (hunk 2 above)
Comment (cloveras) [reply vipps-report-api-faq.md:49]: Thanks, fixed!

File: vipps-report-api-faq.md (hunk 5 above)
Comment (cloveras) [reply vipps-report-api-faq.md:141]: Good idea!

**Pull request merged**